test/utils/bytecode.hpp

```diff
@@ -76,6 +77,20 @@ inline bytecode push(std::string_view hex_data)
     return push(from_hex(hex_data));
 }
 
+bytecode push(evmc_opcode opcode) = delete;
+
+inline bytecode push(evmc_opcode opcode, const bytecode& data)
```

Comment:
I think if you want to reduce the scope of this PR, this helper with its tests could be merged outright.

Reply:
https://github.com/ethereum/evmone/pull/286